README.md

```diff
@@ -365,6 +365,8 @@ Collector exporter packages and types are renamed:
 
 - `serviceName` configuration removed from Collector exporters. Use `service.name` Resource attribute instead.
 
+- Prometheus exporter added suffix `_total` to counter metrics.
```

Comment:
Why blank line above?

Reply:
I'm following the style convention arond the line.